### PR TITLE
media: Remove SpeechDetector::processEPDFrame

### PR DESCRIPTION
**Pull request by junmin-kim**
This api is unnecessary

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [d33f345c7f93ab514e39229c7696a56cccb9243b] - Code Rule Check Result:
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/voice/SpeechDetector.h:39: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/voice/SpeechDetector.h:39: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/voice/SpeechDetector.h:39: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/voice/SpeechDetector.h:39: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/voice/HardwareEndPointDetector.h:28: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/src/media/voice/SoftwareEndPointDetector.h:41: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)

**Comment by seinfra** [comment]
Target : [d33f345c7f93ab514e39229c7696a56cccb9243b] - Code Rule Check (C++) OK.

File: framework/src/media/voice/SoftwareEndPointDetector.cpp [hunk 1]
```diff
@@ -50,26 +50,5 @@ bool SoftwareEndPointDetector::startEndPointDetect(uint32_t timeout)
 	return true;
 }
 
-bool SoftwareEndPointDetector::processEPDFrame(short *sample, int numSample)
```
Comment (Taejun-Kwon) [inline framework/src/media/voice/SoftwareEndPointDetector.cpp]: RecognitionManager calling that function directly, please check it.
We should consider below.
1. sw based epd(check the result of epd for every recording - silence check)
2. hw based epd (silence check based on h/w)
3. hw based epd (callback mechanism)

Comment (junmin-kim) [reply framework/src/media/voice/SoftwareEndPointDetector.cpp]: I'll keep in mind, but do it another PR

**Pull request merged**